src/CreatorToken.sol

```diff
@@ -160,10 +222,18 @@ contract CreatorToken is ERC721Royalty {
     payToken.safeTransferFrom(msg.sender, admin, _totalAdminFee);
   }
 
+  /// @notice Sell a token.
+  /// @dev Sets the minimum accepted proceeds to 0.
+  /// @param _tokenId ID of the token to be sold.
+  /// @return _netProceeds The net proceeds from the sale in USDC.
```

Comment:
Should clarify: "net proceeds from the sale *after fees*"

Reply:
added in https://github.com/showtime-xyz/creator-tokens/commit/3c46721e1c9a0899f54630e2b73028cd5bef159c